### PR TITLE
Ports Ancient bundle from TG

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

Ports https://github.com/tgstation/tgstation/pull/57239 minus all the added defines because I was lazy

## Why It's Good For The Game

see 57239

## Changelog
:cl: YakumoChen
add: A less-than-new Syndicate bundle that reminds you of the good old days when we didn't need all those newfangled traitor items the young-uns get. We had 6 items in the uplink and we had Monkey in rotation and by god we made do.
/:cl:

### CODE REVIEW

**Comment by KrabSpider** [comment]
The bundle contains the following, for anyone that wants to know what the 2006 traitor items were:
> A revolver
>An emag
>Freedom implant 
>Sleepy pen
>Chameleon*
>A pill bottle filled with cyanide pills
>1 telecrystal (for a fail-safe uplink, if I'm reading this correctly?)

**Comment by YakumoChen** [comment]
Not quite. The bundle contains:

-One rusty red toolbox which properly contains the bundle (as a reference to the bundle's age)

All 6 items from an OpenSS13 traitor uplink, circa 2006
-Revolver
-Emag
-Sleepy Pen
-Freedom Implant
-A single 50u cyanide pill
-Chameleon Projector* *The original item was a "cloaking device" which is not in TGcode, and functions similarly.

1 TC meant for a fail-safe uplink, as the original uplink had a self-destruct feature. (fail-safe is an abstract item and I didn't know how to code it.)

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -154,6 +154,20 @@
 			new /obj/item/storage/belt/chameleon(src) // Unique but worth at least 2 tc
 			new /obj/item/card/id/syndicate(src) // 2 tc
 			new /obj/item/chameleon(src) // 7 tc
+			
+		if("ancient") //A kit so old, it's probably older than you. //This bundle is filled with the entire unlink contents traitors had access to in 2006, from OpenSS13. Notably the esword was not a choice but existed in code.
+			/obj/item/storage/toolbox/emergency/old/ancientbundle //Items fit neatly into a classic toolbox just to remind you what the theme is.
+		
+/obj/item/storage/toolbox/emergency/old/ancientbundle/ //So the subtype works
```
Comment (necromanceranne) [inline code/game/objects/items/storage/uplink_kits.dm]:
```suggestion
new /obj/item/storage/toolbox/emergency/old/ancientbundle(src) //So the subtype works
```

A fix from upstream

Comment (Linzolle) [reply code/game/objects/items/storage/uplink_kits.dm]: i'm pretty sure the new was supposed to go to line 159 and not 161.

**Pull request merged**